README.md

```diff
@@ -244,6 +244,12 @@ The question directives accept the following options:
   the key in the submission data. If no key is set, note that automatically added
   keys change when the order and amount of questions is modified.
 
+The `pick-any` directive has a `partial-points` option to award points for partially correct
+submissions. The zero point threshold is when half of the checkboxes are marked correctly, and
+additional correct marks raise the points linearly to the max points.
+A `pick-any` question may have neutral fields, which can be marked with `?`.
```

Comment:
The neutral fields with `?` should be described below in the same paragraph as the notation for correct answers with `*`. The partial points option can be moved there as well.